db.py

```diff
@@ -1,155 +1,173 @@
+import asyncio
 import sqlite3
 
-def database(func):
-    def wrapper(*args,**kwargs):
-        conn = sqlite3.connect('/database/modmail.db')
-        conn.row_factory = sqlite3.Row
-        cursor = conn.cursor()
-        ret = func(cursor, *args, *kwargs)
-        conn.commit()
-        conn.close()
-        return ret
-    return wrapper
-
-@database
-def get_ticket(cursor, ticket_id):
+conn = sqlite3.connect('/database/modmail.db', check_same_thread=False)
```

Comment:
Is changing the connection params for `check_same_thread` to `False` a major problem? Is there a way this can be resolved?

Reply:
from the docs, it looks like as long as the data is serialised, there shouldn't be a problem